modules/platform/nuxeo-platform-task/nuxeo-platform-task-core/pom.xml

```diff
@@ -77,6 +77,10 @@
       <artifactId>nuxeo-platform-userworkspace-types</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>org.nuxeo.ecm.routing</groupId>
+        <artifactId>nuxeo-routing-api</artifactId>
```

Comment:
Routing is using task but not the other way around: the listener should go to `nuxeo-routing-core` with other `DocumentRoute` listeners.